### PR TITLE
Adjustments on Travis

### PR DESCRIPTION
**Pull request by fabricio-aguiar**
- disabled black
- disabled flake8
- generate bindings on pre_before_install.sh
- install it on install.sh

### CODE REVIEW

File: galaxy_ng/app/settings.py [hunk 1]
```diff
@@ -35,3 +35,5 @@
 X_PULP_CONTENT_HOST = "pulp-content"
 X_PULP_CONTENT_PORT = 24816
 X_PULP_CONTENT_PATH_PREFIX = f"/{API_PATH_PREFIX}/v3/artifacts/collections/"
+
+AUTH_USER_MODEL = "galaxy.User"
```
Comment (cutwater) [inline galaxy_ng/app/settings.py]: This change has already been merged.

File: galaxy_ng/tests/functional/api/__init__.py [hunk 2]
```diff
@@ -0,0 +1,2 @@
+# coding=utf-8
```
Comment (cutwater) [inline galaxy_ng/tests/functional/api/__init__.py]: Minor: For python 3 source file encoding is utf-8 by default, therefore this comment is redundant.

**Comment by fabricio-aguiar** [comment]
I believe docker image build process is working again,
I don't know why travis is not running on this PR, but I got it working here:
https://travis-ci.com/github/fabricio-aguiar/galaxy_ng/builds/152854697

the trick I did to get it running on [pulplift](https://github.com/pulp/pulplift#pulplift) was:
build the bindings locally
```
➜  galaxy_ng git:(travis2) cd bindings    
➜  bindings git:(travis2) ./build.sh
```
put the `local.dev-config.yml` file on pulplift dir:
```yaml
# Options
pulp_devel_supplement_bashrc: true
pulp_default_admin_password: password
pulp_install_plugins:
  pulp-ansible:
    source_dir: "/home/vagrant/devel/pulp_ansible"
  galaxy-pulp:
    source_dir: "/home/vagrant/devel/galaxy_ng/bindings/galaxy-pulp"
  galaxy-ng:
    source_dir: "/home/vagrant/devel/galaxy_ng"
  # pulp-certguard:
  #   source_dir: "/home/vagrant/devel/pulp-certguard"
  # pulp-cookbook:
  #   source_dir: "/home/vagrant/devel/pulp_cookbook"
  # pulp-deb:
  #   source_dir: "/home/vagrant/devel/pulp_deb"
  # pulp-container:
  #   source_dir: "/home/vagrant/devel/pulp_container"
  # pulp-gem:
  #   source_dir: "/home/vagrant/devel/pulp_gem"
  # pulp-maven:
  #   source_dir: "/home/vagrant/devel/pulp_maven"
  # pulp-python:
  #   source_dir: "/home/vagrant/devel/pulp_python"
  # pulp-rpm:
  #   source_dir: "/home/vagrant/devel/pulp_rpm"
  # You can specify a folderpath to pulp.pulp_rpm_prerequisites instead of
  # just uncommenting this role name, but note that it is a path on the
  # pulplift host, not the guest VM like all the other paths.
    # prereq_role: "pulp.pulp_rpm_prerequisites"
  pulp-file:
    source_dir: "/home/vagrant/devel/pulp_file"

# Vagrant source install Required
pulp_user: "vagrant"
developer_user: "vagrant"
developer_user_home: "/home/vagrant"
pulp_source_dir: "/home/vagrant/devel/pulpcore"
# pulp_use_system_wide_pkgs: true
pulp_settings:
  secret_key: "unsafe_default"
  content_origin: "http://{{ ansible_fqdn }}"
pulp_webserver_server: 'nginx'
```

File: unittest_requirements.txt [hunk 3]
```diff
@@ -0,0 +1 @@
+mock
```
Comment (cutwater) [inline unittest_requirements.txt]: Why `mock` is required?
Starting from python 3.3 this package is included into standard library as `unittest.mock`.

File: .travis/pre_before_install.sh [hunk 4]
```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
+
+bash ./bindings/build.sh
+pip install -e ./bindings/galaxy-pulp
```
Comment (cutwater) [inline .travis/pre_before_install.sh]: :+1:

File: unittest_requirements.txt (hunk 3 above)
Comment (fabricio-aguiar) [reply unittest_requirements.txt]: it came from template:
https://github.com/pulp/plugin_template/blob/master/templates/test/unittest_requirements.txt.j2
I'll remove it there

**Pull request merged**